### PR TITLE
Add Config Option for CE7 and CSN10 (#31)

### PR DESCRIPTION
**Pull request by enny1170**

### CODE REVIEW

File: MYSBootloader.c [hunk 1]
```diff
@@ -64,13 +64,14 @@
 #define BOOTLOADER_COMMANDS					// Enable low level bootloader commands
 
 // SPI bus setting *****************************************************************************************************
-#define SPI_PINS_CE9_CSN10							
+//define SPI_PINS_CE9_CSN10
```
Comment (tekka007) [inline MYSBootloader.c]: Default configuration should be enabled

File: MYSBootloader.c [hunk 2]
```diff
@@ -64,13 +64,14 @@
 #define BOOTLOADER_COMMANDS					// Enable low level bootloader commands
 
 // SPI bus setting *****************************************************************************************************
-#define SPI_PINS_CE9_CSN10							
+//define SPI_PINS_CE9_CSN10							
 //define SPI_PINS_CSN7_CE8
+#define SPI_PINS_CE7_CSN10
```
Comment (tekka007) [inline MYSBootloader.c]: Comment this line, see above

File: MYSBootloader.c [hunk 3]
```diff
@@ -64,13 +64,14 @@
...
 
 // LED settings ********************************************************************************************************
 #define LED_DDR     DDRB
 #define LED_PORT    PORTB
-#define LED_PIN		PINB5
+#define LED_PIN		PB1
```
Comment (tekka007) [inline MYSBootloader.c]: Default setting should be used here, you can add PINB5 as comment

File: Makefile [hunk 4]
```diff
@@ -1,8 +1,8 @@
 PROJECT = MYSBootloader
 
 MCU = atmega328p
-CLK = 16000000L
-BAUDRATE = 115200
+CLK = 8000000
```
Comment (tekka007) [inline Makefile]: Default settings should be used

File: Makefile [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
+BAUDRATE = 57600
```
Comment (tekka007) [inline Makefile]: Default settings should be used, see above

**Review by tekka007 — CHANGES_REQUESTED** [review]
See comments

**Comment by enny1170** [comment]
changes done. please review again

**Review by tekka007 — APPROVED** [review]

**Pull request merged**